### ISSUE TITLE
Does not work with async functions

### ISSUE THREAD

**Issue by holtc** [heavenshell/vim-jsdoc#99]
Running `:JsDoc` on the following function does not work properly. 

`const doSomething = async (param1, param2, param3) => {}` turns into:

```
/**
 * @type = {}
 */
```

Instead, I believe it should say the below (like it does without the `async`):
```
/**
 * doSomething.
 * @param {} param1
 * @param {} param2
 */
```

**Comment by heavenshell** [comment]
@holtc Thx for reporting.
This is very annoying behavior.
Currently vim-jsdoc can't judge `async() {}` is method or function(AST can judge but Vim script can't).
I'll try to fix this but bit hard.

Just workaround solution, you can do `<Shft-v>` at  `const doSomething` line and type `:'<,'>JsDoc`

**Comment by heavenshell** [comment]
@holtc fixed and release as 2.1.1
Thank you for reporting this!
